README.md

```diff
@@ -56,7 +56,7 @@ The add-in provides the ability to create geodesic features such as lines, circl
 * Running the add-in
 	* To run from a stand-alone deployment
 		* ArcMap
-			* Install the add-in from the application folder by double clicking the **ArcMapAddinDistanceAndDirection.esriAddIn** file.
+			* Install the add-in from the latest Release on Github (https://github.com/Esri/distance-direction-addin-dotnet/releases)
```

Comment:
The plan is to include the add-ins in the release as a file/attachment.  We'll discuss more in SCRUM today.  Just wanted to update the readme to reflect that the Application folder is no longer there.